eng/submit-source-build-release-pr.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+set -euxo pipefail
+
+time="$(date +%s)"
+
+print-help ()
+{
+   echo "Submits a PR that updates the source-build artifacts version and
+global.json dotnet version. Depends on jq, git, and the gh cli. The gh cli also
+must be authenticated, otherwise the GH_TOKEN environment variable must be
+set to a GitHub token that has permission to edit the forked repo and submit
+a PR in the target repo."
+   echo
+   echo "Syntax: ./submit-source-build-release-pr.sh --target-repo dotnet/installer --fork-repo dotnet-sb-bot/installer --sdkVersion 6.0.101"
+   echo "options:"
+   echo "--targetRepo, -t      The repo to submit the PR to"
+   echo "--forkRepo, -f        The repo to submit the PR from"
+   echo "--sdkVersion, -v      The .NET SDK version to update to"
+   echo "--title, -T           The title of the PR"
+   echo "--body, -B            The body of the PR"
+   echo "--targetBranch, -b    (Optional) branch to submit the PR to, calculated automatically otherwise"
+   echo "--globalJson, -g      (Optional) path to the global.json file to update"
```

Comment:
What's the scenario for having this optional parameter as well as versionsProps given the script creates a clone of the repo?

Reply:
I have these parameters because my test repo has both files in the root directory.